package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "5.1.3",
+  "version": "5.1.5",
```

Comment:
I don't know why GH isn't showing the latest version (per the main branch) on the left side of the diff, but your change here is indeed correct.

After this PR is merged, please confirm that your change is reflected in main branch.

Reply:
will do